v2/features/farms/FarmsTableRowHeader.tsx

```diff
@@ -160,24 +183,31 @@ const FarmsTableRowHeader: FC<Props> = ({ jar, simple, open }) => {
           "rounded-tl-xl flex items-center",
         )}
       >
-        <div className="w-9 h-9 rounded-full border-3 border-gray-outline mr-3">
-          <Image
-            src="/alchemix.png"
-            className="rounded-full"
-            width={200}
-            height={200}
-            layout="responsive"
-            alt={jar.depositToken.name}
-            title={jar.depositToken.name}
-          />
-        </div>
+        <TokenIcon
+          src={
+            FARM_LP_TO_ICON[
+              jar.contract as keyof typeof FARM_LP_TO_ICON
+            ]
+          }
+        />
         <div>
           <p className="font-title font-medium text-base leading-5 group-hover:text-green-light transition duration-300 ease-in-out">
             {jar.depositToken.name}
           </p>
           <p className="italic font-normal text-xs text-gray-light">
-            {jar.protocol}
+            {`${jar.protocol} on ${formatNetworkName(jar.chain, allCore)}`}
```

Comment:
This doesn't respect i18n. I was proposing something like this:
<img width="358" alt="Screen Shot 2022-01-14 at 14 16 33" src="https://user-images.githubusercontent.com/7811733/149466999-1546fc1c-f5dd-4c8c-b94b-0a093495fb55.png">

Reply:
@janklimo sorry about that! still have some work to do here. I will work on this in the morning.

Reply:
Sorry about that, I still have some more work to do here. I'll fix it in the morning